APIManager.js

```diff
@@ -0,0 +1,80 @@
+class APIManager {
+  constructor() {
+    this.data = {
+      userData: '',
+      baconData: '',
+      kenyaData: '',
+      pokemonData: '',
```

Comment:
you don't need that, you could declare data just as an empty object (this.data = {}) and each method would create the key-value pairs